### PR TITLE
Add "HTML partials/includes WebComponents-way" article

### PR DESCRIPTION
**Pull request by tomalec**
An article that describes how to use the power of all Web Components specs combined, to build modular, dynamically composable apps. The way to use Web Platform as a replacement to framework-specific and/or server-side templating engines.

//cc @zenorocha, @Nevraeka

### CODE REVIEW

File: src/documents/articles/html-partials-includes-webcomponents-way.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: HTML partials/includes WebComponents-way
+authors: [tomalec]
+date: 2016-09-18
+original_date: 2016-09-18
+link: http://starcounter.io/html-partialsincludes-webcomponents-way/
+category: articles
+layout: single
+tags: ['Partial', 'include', 'import', 'template', 'binding', 'SPA', 'Web Platform', 'HTML Imports', 'Templates', 'Custom Elements']
```
Comment (Nevraeka) [inline src/documents/articles/html-partials-includes-webcomponents-way.md]: tags should be limited to 'HTML Imports'

**Comment by tomalec** [comment]
Thanks: ) , tags updated as you said 
However, I'd say that at least `Templates` also applies :sunglasses:

File: src/documents/articles/html-partials-includes-webcomponents-way.md (hunk 1 above)
Comment (tomalec) [inline src/documents/articles/html-partials-includes-webcomponents-way.md]: done

**Comment by Nevraeka** [comment]
I had to reduce the summary text because it was too much but it's merged now. Thank you @tomalec

**Comment by tomalec** [comment]
Thanks a lot! :)

BTW, do you know why my description didn't get wrapped with `<p>` as the others at http://webcomponents.org/articles/? It results in inconsistent line-spacing.

**Pull request merged**